wfe2/wfe.go

```diff
@@ -2329,16 +2339,20 @@ func (wfe *WebFrontEndImpl) RenewalInfo(ctx context.Context, logEvent *web.Reque
 	}
 
 	if len(result.Incidents) > 0 {
-		// Since IncidentsForSerial() only returns enabled incidents we can
-		// assume that this serial is impacted by an ongoing incident.
-		ri := core.RenewalInfoImmediate(wfe.clk.Now())
+		sendRI(core.RenewalInfoImmediate(wfe.clk.Now()))
+		return
+	}
 
-		err = wfe.writeJsonResponse(response, logEvent, http.StatusOK, ri)
-		if err != nil {
-			wfe.sendError(response, logEvent, probs.ServerInternal("Error marshalling renewalInfo"), err)
-			return
-		}
-		setDefaultRetryAfterHeader(response)
+	// Check if the serial is revoked, in which case it should be replaced now.
```

Comment:
```suggestion
	// Check if the serial is revoked, in which case the client should replace it now.
```

The present tense was confusing and made me think "wait why would it already have been replaced now?"